File: libraryCoreJava/src/main/java/com/juqitech/moretickets/core/http/HttpStatusCode.java
package com.juqitech.moretickets.core.http;

/**
 * @author zhanfeng
 * @date 2019-06-05
 * @desc
 */
public class HttpStatusCode {

    public static final int NETWORK_EXCEPTION = -1000;
    public static final int NOT_NETWORK_EXCEPTION = -1;
    public static final int PARSE_EXCEPTION = -2;
    public static final int EXCEPTION = -10;
    public static final int SUCCESS = 200;
    public static final int SESSION_EXPIRED = 1005;
    public static final int LOGIN_EXPIRED = 1006;
    public static final int REFRESH_SESSION_FAILURE = -1005;

}

File: app/src/main/java/com/theone/mvvm/base/constant/HttpStatusCode.kt
package com.shownow.shownow.base.constant

/**
 * @author zhanfeng
 * @date 2019-06-05
 * @desc
 */
object HttpStatusCode {

    val NETWORK_EXCEPTION = -1000
    val NOT_NETWORK_EXCEPTION = -1
    val PARSE_EXCEPTION = -2
    val EXCEPTION = -10
    val SUCCESS = 200
    val SESSION_EXPIRED = 1003
    val LOGIN_EXPIRED = 1006
    val REFRESH_TOKEN_EXPIRED = 1042
    val REFRESH_SESSION_FAILURE = -1005

}